docs/Telegram.md

```diff
@@ -1,73 +1,68 @@
-before chatting.
-
-#### Notifications
-
-This is a relatively high traffic channel (replacing IRC, which is no
-longer active). It's recommended that if using a phone app you disable
-notifications for the group (you will still receive notifications if
-someone sends you a PM, or if someone addresses or mentions you
-specifically).
-
-### Group Channels
-
-**Note we were using <http://protect-mylinks.com/addlinks> to avoid spam bots adding
-themselves to the channels, but it looks like they've started using annoying adds so that's been turned off**
-
-Several groups that look after parts of the hackspace have specific
-channels on telegram so they don't clog up the main channel.
-
--   Laser Training Group's [Telegram
-    group](https://t.me/joinchat/D5l4WUNG_uyg6YB0PYikRA) for
-    members who want training on the laser
--   Metalworking [Telegram
-    group](https://t.me/joinchat/B8-OC1MTETWTM8vBexJOag)
--   [Team Woodwork](Team_Woodwork "wikilink")'s [Telegram
-    group](https://t.me/joinchat/AYtZgkk7n1MqvkN9N2fmsA)
--   3d Printing [Telegram
-    group](https://t.me/joinchat/DZNJNRJimIP7XoyvDArnUg)
--   Hacman CNC [Telegram
-    Group](https://t.me/joinchat/AYtZgkh3BSRgpoA9LvQ3Hg)
--   Hacman Announcements [Telegram
-    Group](https://t.me/hackspaceannounce)
--   Hacman Games Night [Telegram
-    Group](https://t.me/joinchat/Ef01rUOXtWoUe3NHzQ-9QQ)
-
-TODO Expired Links
-
--   Notifications from the Hackscreen, including who's been in [Telegram
-    group](https://t.me/joinchat/Ef01rU1SrP9xJNmwXjHtWA)
--   [Team Procurement](Team_Procurement "wikilink")'s [Telegram
-    group](https://t.me/joinchat/D5l4WUPaGRQovcFCLd1X7g)
--   [Team Documentation](Team_Documentation "wikilink")'s [Telegram
-    group](https://t.me/joinchat/Agaj2UBrygQwTBBHDW8jyA)
--   [Team Event](Team_Events "wikilink")'s [Telegram
-    group](https://t.me/joinchat/E4DcrUDWGvEzlsmJHNNkuQ)
--   Space Bikes [Telegram
-    Group](https://t.me/joinchat/AoAv41LtXqjULCvqsbGwrA)
-
-
-
-
-Groups exist for the following: [laser
-maintenance](Team_Laser "wikilink"), security, board, etc. These groups
-require training/election to join so please ask in the main channel if
-you are interested.
+We have a number of group chats in [Telegram](https://telegram.org) that we encourage members to use to keep in touch, help each other out, and stay up to date with news and announcements.
+
+Some of these group chats are used to coordinate training for certain pieces of equipment.
+
+Telegram has a number of clients for various platforms:
+
+- [Web client](https://telegram.org/dl/webogram) 
+- [Desktop (Windows, Linux, macOS)](https://desktop.telegram.org/)
+- [Android](https://telegram.org/dl/android)
+- [iOS](https://telegram.org/dl/ios)
+- [Windows Phone](https://telegram.org/dl/wp)
+
+
+You will need a phone number to sign up for Telegram. This is kept private and hidden from other Telegram users by default.
```

Comment:
I love the more detailed explanation of Telegram. One thing to add maybe as I know a few people (me included) got really pissed off that Telegram blabbed to everyone that I joined:


```suggestion
You will need a phone number to sign up for Telegram. This is kept private and hidden from other Telegram users by default.

Please note that when you join Telegram, people with your phone number in their address book will be notified that you joined Telegram. 
```